ontology/doc/posible_changes.md

```diff
@@ -0,0 +1,17 @@
+## DONE: Remove Mobile Deployment
+
+`MobileDeployment` is not really needed to capture activities involving mobile platforms. A better solution is proposed in the draft notes for UAV datasets where sensors are deployed to a mobile platform which then participates in Environmental Monitoring Activities.
+
+With this change, there may then be a case for collapsing `StaticDeployment` into `Deployment`.
+
+# DONE: Remove fdri:Agent and its subclasses
+
+Rather than remove the FDRI ontology classes we opted to instead just root the FDRI Agent hierarhcy as a subclass of prov:Agent.
+
+# DONE: Remove Catchment and Region types
+
+These are now terms in a FacilityGroupType concept scheme
+
+# Make InternalDataProcessingConfiguration a subclass of ConfigurationValueSeries
+
+An InternalDataProcessingConfiguration has hasCurrentConfiguration and hadConfiguration which are mirrored in ConfigurationValueSeries as hasCurrentValue and hadValue. Thus an  InternalDataProcessingConfiguration could be seen as a ConfigurationValueSeries where the value item is a ConfigurationItem (or a set of ConfigurationItems, although I do not believe we are currently using this).
```

Comment:
Thanks would make sense. Are both in active use at the moment? If we only use `InternalDataProcessingConfiguration` atm then could rename the properties for `ConfigurationValueSeries` to match current usage to minimise disruption of the change?

Reply:
That would be possible. It would also be possible to do the renaming only in the OWL ontology and keep the existing names in the recordspec as the property short code in recordspec is separate from the property IRI. Though possibly that would be confusing for future us.
However, I think the cleanest approach would be to do the subclassing as noted here if that's OK with the devs. For that reason I think I might put this change in as a separate ticket that we can discuss with the devs.